<launch>
    <arg name="ns" default="leap"/>
    <arg name="hands_topic" default="hand_info"/>
    <node pkg="leap_client" type="leap_client_node" name="$(arg ns)">
        <param name="refresh_rate" type="double" value="200.0"/>
        <param name="hands_topic" type="str" value="$(arg hands_topic)"/>
    </node>
    <node pkg="leap_client" type="leap_viz" name="leap_viz">
        <param name="hands_topic" type="str" value="/$(arg ns)/$(arg hands_topic)"/>
    </node>
    <node pkg="tf" type="static_transform_publisher" name="world_tf" args="0 0 0 0 0 0 world $(arg ns) 100" />
</launch>
